tests/integration/test_benchmark.py

```diff
@@ -54,7 +54,7 @@ def test_benchmark_single_table_basic_synthsizers():
     ] == quality_scores.index.tolist()
 
 
-@pytest.mark.skipif(sys.platform.startswith('darwin'), reason='Test not supported on github MacOS')
+@pytest.mark.skip('`create_sdv_synthesizer_variant` should only work with SDV synthesizers.')
```

Comment:
Let me know if this makes sense. In this test, we were creating a variant of RealTabFormer with `create_sdv_synthesizer_variant`, which I found confusing.